xarray/tests/test_backends.py

```diff
@@ -1438,6 +1438,265 @@ class TestPyNioAutocloseTrue(TestPyNio):
     autoclose = True
 
 
+@requires_rasterio
+class TestRasterio(CFEncodedDataTest, Only32BitTypes, TestCase):
```

Comment:
I don't think there's any advantage to inheriting from the other test methods, given that this isn't an actual datastore. I think this can just inherit from `TestCase` (then you can delete the first two methods).